Misc/NEWS.d/next/Library/2020-07-10-14-33-52.bpo-41270.pjUqyd.rst

```diff
@@ -0,0 +1 @@
+Made NamedTemporaryFile its own iterator to mimic file objects.
```

Comment:
Please use rST markup.
```rst
:func:`tempfile.NamedTemporaryFile`
```
You may be asked to reword this news entry.  It would be nice to have more details.

Reply:
reworded